### PR TITLE
OSSMDOC-586: Service Mesh 2.2.1 Release Notes

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-586

Link to 2.2.1 docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-586/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-version-2-2-1

Link to 2.1.4 docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-586/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-2-1-4

Link to 2.0.10 docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-586/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-2-0-10

Additional information:
Eng review: jwendell
QE review: pbajjuri20
Peer review: jstickler

### CODE REVIEW

File: modules/ossm-rn-new-features.adoc [hunk 1]
```diff
@@ -128,6 +174,7 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 |1.30.2
 
 |Kiali
+>>>>>>> 78c8521e9f (OSSMDOC-586: Service Mesh 2.2.1 Release Notes)
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: @jwendell thanks for pointing this out. The update has been submitted.

**Review by jwendell — APPROVED** [review]

**Comment by pbajjuri20** [comment]
It looks good to me.

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-rn-known-issues.adoc [hunk 2]
```diff
@@ -34,24 +34,13 @@ These are the known issues in {SMProductName}:
...
 * https://issues.redhat.com/browse/OSSM-1655[OSSM-1655] Kiali dashboard shows error after enabling mTLS in `SMCP`.
 +
 After enabling the `spec.security.controlPlane.mtls` setting in the SMCP, the Kiali console displays the following error message `No subsets defined`.
 
 * https://issues.redhat.com/browse/OSSM-1396[OSSM-1396] If a gateway resource contains the `spec.externalIPs` setting, instead of being recreated when the `ServiceMeshControlPlane` is updated, the gateway is removed and never recreated.
 
-* https://issues.redhat.com/browse/OSSM-1211[OSSM-1211]
-Configuring Federated service meshes for failover does not work as expected.
-+
-The Istiod pilot log displays the following error: `envoy connection [C289] TLS error: 337047686:SSL routines:tls_process_server_certificate:certificate verify failed`
```
Comment (JStickler) [inline modules/ossm-rn-known-issues.adoc:51]: Can we retain this error message in the Fixed Issues section?  That way customers who haven't upgraded and run into this issue will know that this release addresses this.

Comment (rh-tokeefe) [reply modules/ossm-rn-known-issues.adoc:51]: I've added this back and squashed the job.

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**